### DISCUSSION TITLE
How can I use Scheme URI in markdown?

### DISCUSSION THREAD

**Discussion by AkagiYui** [facebook/docusaurus#8715] [Q&A] [answered]
```markdown
Click [this](ms-settings:themes) open theme settings
```
I want to use scheme URI like this in markdown.
But after building, the link is redirected to path in docusaurus.
What can I do to make it work?

**Comment by slorber** [comment, marked as answer]
> But after building, the link is redirected to path in docusaurus.

This is because we use React Router, and it's a React Router (v5) behavior when using a link with custom sceme

https://codesandbox.io/s/yv87xknz7x?file=/src/Routes.js

![CleanShot 2023-03-01 at 17 15 46@2x](https://user-images.githubusercontent.com/749374/222198468-0373ba62-c04d-40ee-ba46-bf78a80e1b29.png)

I think we can consider it a bug and should not use the React Router link once a non-http(s) scheme is provided? 🤷‍♂️
